lib/resources/r4/types.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
So this file is required by https://github.com/cerner/fhir.cerner.com/blob/master/lib/definition_table_generator.rb#L33 to generate a yaml table. Not sure what exactly a minimum set might look like but I'll play around with it some.

Reply:
There's a bit of info about how this works/what it is for on the readme: https://github.com/cerner/fhir.cerner.com

Reply:
I see thanks! Is there a preference on which to use for the url? either the resource specific link like http://hl7.org/fhir/r4/encounter-definitions.html#Encounter.identifier or the field dataType link http://hl7.org/fhir/r4/datatypes.html#Identifier?

Reply:
assuming datatypes since that's what DSTU2 used

Reply:
Added a minimum set of datatype urls to r4/types.yaml and removed the specific urls from the encounter yaml: https://github.com/cerner/fhir.cerner.com/pull/239/commits/512b564a5cb8f711b3cb4e115e8ac24b45f62fcb